webpack.config.js

```diff
@@ -3,7 +3,8 @@ var path = require("path");
 
 var entries = {
   background: "./webextension/background.js",
-  manage: "./webextension/manage.js"
+  manage: "./webextension/manage.js",
+  "babel-polyfill"
```

Comment:
@linuxwolf did I get this merged in properly to correctly include babel-polyfill?